test/e2e-tests.sh

```diff
@@ -130,7 +114,7 @@ ko apply -f config/provisioners/in-memory-channel/in-memory-channel.yaml
 wait_until_pods_running knative-eventing
 
 # Publish test images
-publish_test_images
+bash $(dirname $0)/upload-test-images.sh e2e
```

Comment:
No need to explicitly use bash here.

Reply:
oh, I can call the script directly. Using `sh` was giving me compilation error.